pygsti/objects/operation.py

```diff
@@ -986,6 +988,45 @@ def transform_inplace(self, s):
         Smx = s.transform_matrix
         Si = s.transform_matrix_inverse
         self.set_dense(_np.dot(Si, _np.dot(self.to_dense(), Smx)))
+    
+    def spam_transform_inplace(self, s, typ):
```

Comment:
What are these spam_transform_inplace updates needed for?  Is it possible to save them until later to ease merging with other planned updates?

Reply:
I believe these are to enable gauge transformations. These are available in LindbladOp for the LindbladSPAMVec/POVM, so I piped it up to the base class.

However, these can definitely be saved for later. I've removed them for now.